### PR TITLE
schema: Use schema repository via submodule

### PR DESCRIPTION
**Pull request by bjhargrave**
This avoids the need to duplicate the schema files in this repository.

### CODE REVIEW

**Review by ckadner — APPROVED** [review]
Nice!  LGTM

**Review by lehors — APPROVED** [review]
LGTM. I hope it works (I didn't have a chance to try it yet).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -59,6 +59,15 @@ To contribute to this repo, you'll use the _Fork and Pull_ model common in many
 - For details on the local process, check out the [GitHub flow](https://docs.github.com/en/get-started/using-github/github-flow) documentation from GitHub and [The GitHub Workflow Guide](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md) documentation from Kubernetes.
 - For details on contributing using the GitHub webpage UI, see [Contributing using the GH UI](docs/contributing_via_GH_UI.md).
 
+This repository uses [submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) to incorporate the [taxonomy schema](https://github.com/instructlab/schema.git).
+So, when using a local clone of this Git repository locally, be sure to use the [`--recurse-submodules`](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---recurse-submodulesltpathspecgt) option on the `git clone` command and the `git pull` command when pulling updates from the remote repository.
```
Comment (lehors) [inline CONTRIBUTING.md:63]:
```suggestion
So, when using a local clone of this Git repository, be sure to use the [`--recurse-submodules`](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---recurse-submodulesltpathspecgt) option on the `git clone` command and the `git pull` command when pulling updates from the remote repository.
```

Comment (bjhargrave) [reply CONTRIBUTING.md:63]: It already says "when using a local clone". So a second local seems redundant.

Comment (lehors) [reply CONTRIBUTING.md:63]: I think you read too fast @bjhargrave I suggested *removing* locally because it is indeed redundant.

Comment (lehors) [reply CONTRIBUTING.md:63]: There is a reason the standard policy is to let the commenter mark comments resolved. :-)

**Pull request merged**